pack3d/model.go

```diff
@@ -152,6 +159,40 @@ func (m *Model) ValidChange(i int) bool {
 	return true
 }
 
+func (m *Model) ValidBound(i int, stlsize []fauxgl.Vector, boundsize fauxgl.Vector) bool {
```

Comment:
Some kind of comment/description # True if the passed move it within maximum_packing_area, false in all other cases.